extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -42,13 +42,7 @@ import {viewerForDoc} from '../../../src/viewer';
 import {xhrFor} from '../../../src/xhr';
 import {endsWith} from '../../../src/string';
 import {platformFor} from '../../../src/platform';
-import {
-  importPublicKey,
-  isCryptoAvailable,
-  verifySignature,
-  verifyHashVersion,
-  PublicKeyInfoDef,
-} from './crypto-verifier';
+import {PublicKeyInfoDef, cryptoFor} from '../../../src/crypto';
```

Comment:
Do you actually need to import `PublicKeyInfoDef` if you're just using it for typing?  I _thought_ you could get away without it.  IIRC, you then declare your params like:

```javascript
/** @type {!../../../src/crypto.PublicKeyInfoDef} */
let foo = ...
```